projects/plugins/jetpack/extensions/blocks/map/controls.js

```diff
@@ -120,36 +88,21 @@ export default ( {
 					{
 						value: attributes.markerColor,
 						onChange: value => setAttributes( { markerColor: value } ),
-						label: __( 'Marker Color', 'jetpack' ),
+						label: __( 'Marker', 'jetpack' ),
 					},
 				] }
 			/>
-			<PanelBody title={ __( 'Map Settings', 'jetpack' ) }>
-				<BaseControl
-					__nextHasNoMarginBottom={ true }
+			<PanelBody title={ __( 'Settings', 'jetpack' ) }>
+				<NumberControl
 					label={ __( 'Height in pixels', 'jetpack' ) }
-					id={ `block-jetpack-map-height-input-${ instanceId }` }
-				>
-					<input
-						type="number"
-						id={ `block-jetpack-map-height-input-${ instanceId }` }
-						className="wp-block-jetpack-map__height_input"
-						onChange={ event => {
-							setAttributes( { mapHeight: event.target.value } );
-							// If this input isn't focussed, the onBlur handler won't be triggered
-							// to commit the map size, so we need to check for that.
-							if ( event.target !== event.target.ownerDocument.activeElement ) {
```

Comment:
Similar to the `onHeightChange` comment, I'm not sure if this is needed too since NumberControl should handle the `onBlur` too.